### ISSUE TITLE
Python 3.9 missing in manylinux{1,2014} docker images

### ISSUE THREAD

**Issue by mr-c** [pypa/manylinux#629]
```
$ for VER in 1 2010 2014 ; do docker pull --quiet quay.io/pypa/manylinux${VER}_x86_64 ; docker run -it  quay.io/pypa/manylinux${VER}_x86_64 find / | grep "cp39" || echo quay.io/pypa/manylinux${VER}_x86_64 is missing Python 3.9 ; done
quay.io/pypa/manylinux1_x86_64:latest
quay.io/pypa/manylinux1_x86_64 is missing Python 3.9
quay.io/pypa/manylinux2010_x86_64:latest
/opt/python/cp39-cp39
quay.io/pypa/manylinux2014_x86_64:latest
quay.io/pypa/manylinux2014_x86_64 is missing Python 3.9
```

**Comment by YannickJadoul** [comment]
See #639 & #640

**Comment by mr-c** [comment]
Thank you @YannickJadoul !

**Comment by YannickJadoul** [comment]
Both were merged! Not sure why "closes #629" in the description of #640 didn't work, but I guess you can manually close the issue, then :-)

**Comment by lkollar** [comment]
Thanks @YannickJadoul!
